.azure-pipelines/mgmt.yml

```diff
@@ -6,7 +6,6 @@ pr:
     - BuildToolsForSdk
   paths:
     exclude:
-    - sdk/
```

Comment:
You can undo the changes in this file. I've fixed the triggering on the DevOps side.

Reply:
@shahabhijeet

Reply:
@weshaggard , we can't be blocked by this any longer. I am going to bring this back to get CI build running, as we need to release our SDK.